### PR TITLE
Adds Several Butter-filled Foods

### PR DESCRIPTION
**Pull request by TehZombehz**
:cl: TehZombehz
add: Butter noodles, butter biscuits, buttered toast, and pigs in a blanket can now be crafted using sticks of butter and other ingredients.
/:cl:
butterfoods
Adds butter noodles, butter biscuits, buttered toast, and pigs in a
blanket as craftable foods requiring sticks of butter as an ingredient. If you're unfamiliar with these foods, look them up.
This would give chefs a reason to use butter that's not eating it raw or deep frying it.

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_bread.dm [hunk 1]
```diff
@@ -164,3 +164,24 @@
 	icon = 'icons/obj/food/food.dmi'
 	icon_state = ""
 	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/butteredtoast
+	name = "buttered toast"
+	desc = "Butter lightly spread over a peice of toast."
```
Comment (MrPerson) [inline code/modules/food_and_drinks/food/snacks_bread.dm]: That's not how you spell "piece"

Comment (TehZombehz) [inline code/modules/food_and_drinks/food/snacks_bread.dm]: It was 4AM and I was tired.

Comment (Pascal125) [inline code/modules/food_and_drinks/food/snacks_bread.dm]: It is if you try hard enough.

**Review by Cyberboss — APPROVED** [review]

**Comment by Tacolizard** [comment]
oh my god pigs in a blanket i need them now sooo goood

**Comment by Armhulen** [comment]
Where's my deep fried butter on a stick?

**Pull request merged**